x-pack/plugins/security/server/authorization/disable_ui_capabilities.ts

```diff
@@ -67,20 +68,56 @@ export function disableUICapabilitiesFactory(
       };
     }, {});
 
-  const shouldDisableFeatureUICapability = (
-    featureId: keyof UICapabilities,
-    uiCapability: string
-  ) => {
-    // if the navLink isn't for a feature that we have registered, we don't wish to
-    // disable it based on privileges
-    return featureId !== 'navLinks' || featureNavLinkIds.includes(uiCapability);
+  function isCatalogueItemReferencedByFeatureSet(
```

Comment:
nit: You've got a mix of arrow functions and function declarations here, I'd go with one approach and use it throughout (arrow functions are probably is preferred here unless you require hoisting)

Reply:
Good point! Thanks!